source/arroost/entities/scene.js

```diff
@@ -92,18 +95,7 @@ export class Scene extends Entity {
 			})
 		}, [this.dom.transform.position, this.dom.transform.scale, this.width, this.height])
 
-		const layer = (this.layer = {
-			wire: new Dom({ id: "wire-layer", type: "html" }),
-			timing: new Dom({ id: "timing-layer", type: "html" }),
-			cell: new Dom({ id: "cell-layer", type: "html" }),
-			ghost: new Dom({ id: "ghost-layer", type: "html" }),
-			hud: new Dom({ id: "hud-layer", type: "html" }),
-		})
-
-		this.dom.append(layer.wire)
-		this.dom.append(layer.timing)
-		this.dom.append(layer.cell)
-		this.dom.append(layer.ghost)
+		this.recreateSceneLayers()
```

Comment:
prep for when we'd want to replace the world with a different layer. not used yet